util/simneo.py

```diff
@@ -34,12 +36,21 @@ def __init__(
         enableLimitSwitches: bool = True,
         limitSwitchPolarity: SparkMaxLimitSwitch.Type = SparkMaxLimitSwitch.Type.kNormallyOpen,
     ):
-        self.motor = CANSparkMax(canID, CANSparkMax.MotorType.kBrushless)
+        print(f"Init Spark FLEX with port {canID} with name {name}")
+        self.name = name
+        self.id = canID
+        self.motor = CANSparkFlex(canID, CANSparkFlex.MotorType.kBrushless)
         self.controller = self.motor.getPIDController()
         self.encoder = self.motor.getEncoder()
         self.forwardSwitch = self.motor.getForwardLimitSwitch(limitSwitchPolarity)
         self.reverseSwitch = self.motor.getReverseLimitSwitch(limitSwitchPolarity)
 
+        self._nettableidentifier = f"motors/{self.name}({self.id})"
+        SmartDashboard.putNumber(f"{self._nettableidentifier}/gains/p", pGain)
+        SmartDashboard.putNumber(f"{self._nettableidentifier}/gains/i", iGain)
+        SmartDashboard.putNumber(f"{self._nettableidentifier}/gains/d", dGain)
+        SmartDashboard.putBoolean(f"{self._nettableidentifier}/inverted", isInverted)
```

Comment:
This is more of a Luke question, but do these get placed once or updated every robot cycle? seems like a waste of cpu if it happens constantly

Reply:
once since its in init